src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs

```diff
@@ -5924,7 +5924,7 @@ .maxstack  2
   IL_003a:  brfalse.s  IL_0042
   IL_003c:  ldloc.0
   IL_003d:  call       ""void System.Console.Write(bool)""
-  IL_0042:  ldc.r8     Infinity
+  IL_0042:  ldc.r8     ∞
```

Comment:
Nice :)

Reply:
Is this going to cause problems when run on non "en" cultures (seems the invariant culture isn't being used)?

Reply:
I've filled https://github.com/dotnet/metadata-tools/issues/44 to track. I'm not going to hold up this PR anymore, I need to get it in to address some testing issues in my function-pointers branch.

Reply:
> I've filled [https://github.com/dotnet/metadata-tools/issues/44](https://github.com/dotnet/metadata-tools/issues/44) to track. I'm not going to hold up this PR anymore, I need to get it in to address some testing issues in my function-pointers branch.

Could we run into issues with Spanish CI job?

---
In reply to: [396761882](https://github.com/dotnet/roslyn/pull/42641#discussion_r396761882) [](ancestors = 396761882)

Reply:
Turns out, yes we can. PR'd https://github.com/dotnet/metadata-tools/issues/44 to fix